src/applications/gi/components/search/OnlineClassesFilter.jsx

```diff
@@ -4,34 +4,32 @@ import PropTypes from 'prop-types';
 import { renderLearnMoreLabel } from '../../utils/render';
 import { ariaLabels } from '../../constants';
 
-class OnlineClassesFilter extends React.Component {
-  static propTypes = {
-    onlineClasses: PropTypes.string.isRequired,
-    onChange: PropTypes.func,
-    showModal: PropTypes.func.isRequired,
-  };
-
-  render() {
-    return (
-      <RadioButtons
-        label={renderLearnMoreLabel({
-          text: 'How do you want to take classes?',
-          modal: 'onlineOnlyDistanceLearning',
-          showModal: this.props.showModal,
-          ariaLabel: ariaLabels.learnMore.onlineOnlyDistanceLearning,
-          component: this,
-        })}
-        name="onlineClasses"
-        options={[
-          { value: 'yes', label: 'Online only' },
-          { value: 'no', label: 'In person only' },
-          { value: 'both', label: 'In person and online' },
-        ]}
-        value={this.props.onlineClasses}
-        onChange={this.props.onChange}
-      />
-    );
-  }
+function OnlineClassesFilter({ showModal, onlineClasses, onChange }) {
+  return (
+    <RadioButtons
+      label={renderLearnMoreLabel({
+        text: 'How do you want to take classes?',
+        modal: 'onlineOnlyDistanceLearning',
+        showModal,
+        ariaLabel: ariaLabels.learnMore.onlineOnlyDistanceLearning,
+        component: OnlineClassesFilter,
```

Comment:
Will this still behave the same way?

Reply:
Looks like yes:

![image](https://user-images.githubusercontent.com/2008881/80241119-e86c4180-8617-11ea-81ef-002f5ee75804.png)